cpu/sam0_common/periph/gpio.c

```diff
@@ -65,6 +65,20 @@ typedef enum {
 } gpio_eic_clock_t;
 
 static gpio_isr_ctx_t gpio_config[NUMOF_IRQS];
+
+#if defined(CPU_FAM_SAML10) || defined(CPU_FAM_SAML11)
+static const gpio_t rtc_pins[RTC_NUM_OF_TAMPERS] = {
+    GPIO_PIN(PA, 8), GPIO_PIN(PA, 9), GPIO_PIN(PA, 16),
+    GPIO_PIN(PA, 17)
```

Comment:
Maybe this should go into the respective `periph_cpu.h`.

Reply:
It's also a good idea to keep this file platform agnostic - I'll move it to the respective family headers instead.